ta/gatekeeper/include/gatekeeper_ipc.h

```diff
@@ -0,0 +1,169 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright (c) 2018, Linaro Limited */
+/* Copyright (c) 2017, GlobalLogic  */
+
+#ifndef __GATEKEEPER_IPC_H
+#define __GATEKEEPER_IPC_H
+
+#include <stdint.h>
+#include <string.h>
+
+/*
+ * Please keep this define consistent with TA_UUID variable that defined
+ * in Android.mk file
+ */
+#define TA_GATEKEEPER_UUID { 0x4d573443, 0x6a56, 0x4272, \
+		{ 0xac, 0x6f, 0x24, 0x25, 0xaf, 0x9e, 0xf9, 0xbb} }
+
+/*
+ * GateKeeper message size
```

Comment:
Everywhere in this patch-set where it is a comment:
`s/GateKeeper/Gatekeeper/g` (see correct spelling [here](https://source.android.com/security/authentication/gatekeeper))

Reply:
fixed